### PR TITLE
CSVerifier fixtures

### PR DESCRIPTION
**Pull request by madlabman**

### CODE REVIEW

File: test/CSVerifier.t.sol [hunk 1]
```diff
@@ -48,8 +45,7 @@ contract CSVerifierTest is Test {
             gIHistoricalSummaries: pack(0x0, 0), // We don't care of the value for this test.
             gIFirstWithdrawal: pack(0xe1c0, 4),
             gIFirstValidator: pack(0x560000000000, 40),
-            // TODO: Fix proofs and set Slot.wrap(DENEB_FORK_EPOCH * 32)
-            firstSupportedSlot: Slot.wrap(1357000)
+            firstSupportedSlot: Slot.wrap(0xa5)
```
Comment (dgusakov) [inline test/CSVerifier.t.sol:48]: Can we have explanatory comment here?

File: test/fixtures/CSVerifier/generator.mjs [hunk 2]
```diff
@@ -0,0 +1,169 @@
+import fs from "fs";
```
Comment (dgusakov) [inline test/fixtures/CSVerifier/generator.mjs:1]: Why is file extension mjs? It is not minimised

Comment (madlabman) [reply test/fixtures/CSVerifier/generator.mjs:1]: It's to make node to think of the script as of a module for imports to work.

File: script/gindex.mjs [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { concatGindices } from "@chainsafe/persistent-merkle-tree";
+import { ssz } from "@lodestar/types";
+
+const BeaconState = ssz.deneb.BeaconState;
+const BeaconBlock = ssz.deneb.BeaconBlock;
+
+{
+  const Withdrawals = BeaconBlock.getPathInfo(["body", "executionPayload", "withdrawals"]).type;
+  const nav = BeaconState.getPathInfo(["latestExecutionPayloadHeader", "withdrawalsRoot"]);
```
Comment (vgorkavenko) [inline script/gindex.mjs:9]: `nav` quite misleading

File: script/gindex.mjs
Comment (vgorkavenko) [inline script/gindex.mjs:1]: might be helpful to have some link to the spec that we do the things properly

**Review by vgorkavenko — COMMENTED** [review]
Would be great to have some section in README with explainers why and when we must\can use the scripts\fixtures manually

File: script/gindex.mjs
Comment (madlabman) [reply script/gindex.mjs:1]: `getPathInfo` throws if we try to traverse a structure via non-existent path.

**Review by vgorkavenko — DISMISSED** [review]

**Review by vgorkavenko — DISMISSED** [review]

**Review by dgusakov — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**